scripts/lxd-images

```diff
@@ -734,6 +742,11 @@ if __name__ == "__main__":
                                       help=_("Keep this image up to date"))
     parser_import_ubuntu.set_defaults(func=import_ubuntu)
 
+    # # legacy LXC
+    parser_import_lxc = parser_import_subparsers.add_parser(
+        "lxc", help=_("Error message for legacy LXC images"))
```

Comment:
Remove the help message and change:

    parser_import_subparsers = parser_import.add_subparsers(dest="source")

for

    parser_import_subparsers = parser_import.add_subparsers(dest="source", metavar="{busybox,ubuntu}")

That should make the lxc subparser vanish from the help.